src/web/components/SlotBodyEnd/BrazeEpic.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useEffect, useState } from 'react';
+import * as emotion from 'emotion';
+import * as emotionCore from '@emotion/core';
+import * as emotionTheming from 'emotion-theming';
```

Comment:
Is this pattern of importing emotion needed? We're about to migrate to Emotion 11 and I'm not sure how this fits into that

Reply:
This relates to how these core deps are shared with dynamically imported components via `window.guardian.automat` (also the case for reader revenue components). I think it's on the roadmap for the Emotion upgrade (I've chatted to Gareth and William about this in recent weeks).

Reply:
Sounds good